StylusDeepDark.user.css

```diff
@@ -616,13 +616,16 @@
 	}
 
 	/*Stylus popup fixes */
+	/* Use opacity to solve disabled entries see #18 thx Mottie :)*/
 	#stylus-popup .disabled .style-name
 	{
-		color: var(--hover-background) !important;
+		opacity: .7;
```

Comment:
@Mottie is this opacity level OK with you?

Reply:
Yeah, that's better! 😺